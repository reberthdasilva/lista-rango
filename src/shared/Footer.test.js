import React from "react";
import Footer from "./Footer";
import renderer from "react-test-renderer";

test("Testing Footer", () => {
  const component = renderer.create(<Footer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
