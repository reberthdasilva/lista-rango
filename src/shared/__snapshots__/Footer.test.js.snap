// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer 1`] = `
<footer
  className="footer"
>
  Footer
</footer>
`;
